packages/web-twig/src/Resources/components/Dropdown/Dropdown.twig

```diff
@@ -0,0 +1,16 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _isOpen = props.isOpen | default(false) | boolprop -%}
```

Comment:
State is handled by JS and the `is-open` class is reserved for JS too. Therefore we don't need this option.

Reply:
Good point, thanks.